flink-java/src/main/java/org/apache/flink/api/java/utils/DataSetUtils.java

```diff
@@ -141,7 +141,7 @@ public void mapPartition(Iterable<T> values, Collector<Tuple2<Long, T>> out) thr
 	 *  <li> c is shifted by n bits where n = log2(number of parallel tasks)
 	 * 	<li> to create a unique ID among all tasks, the task id is added to the counter
 	 * 	<li> for each record, the resulting counter is collected
-	 * </ul>
+	 * </ul>.
```

Comment:
This will probably render weird in the final javadocs. Let's reword the first sentence slightly; i.e. replace "the following way:" -> "as described below."